README.md

```diff
@@ -44,21 +46,94 @@ Check the provider documentation for provider-specific files and configuration t
 Start the deployment process
 
 ```
-$ docker run -ti -v (path to your directory):/root/data docker.io/osevg/openshifter create cluster01
+$ docker run -ti -v `pwd`:/root/data docker.io/osevg/openshifter create cluster01
 ```
 
 ## Provider documentation
 
 ### GCE
 
-1. Create a Google Cloud Patlform Project
-1. Enable Compute Engine API
-1. Enable DNS API
-1. Create Cloud DNS Zone (e.g., openshift-mydomain-com for openshift.mydomain.com suffix)
-1. Create a new Service Account with Project Owner role, and furnish a new JSON key
+Following are instructions required to get OpenShift working on top Google Cloud Platform (GCP).
+
+First, you need to get a GCP account and create a project for running OpenShift on it.
+Creating the project can be done using the GCP user interface or via the [`gce/create-project-gce.sh`](gce/create-project-gce.sh) script.
+The instructions for using the user interface are not available yet, so the best would be to use the script.
+
+To run the script, you just need to provide a name and the email registered with your GCP account:
```

Comment:
should mention that the script requires `gcloud` tool installed on users machine